### PR TITLE
KIN-220 Renaming of the emails, email_types and users tables

### PR DESCRIPTION
**Pull request by CarlosJCD**
[KIN-220](https://loopcrack.atlassian.net/browse/KIN-220) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

Standarize the database to a single naming convention
### Solution

Renaming the existing table to match the naming convention

## Resources

- [Naming conventions document](https://loopcrack.atlassian.net/wiki/spaces/Kinub/pages/14188560)

[KIN-220]: https://loopcrack.atlassian.net/browse/KIN-220?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: backup/Database.sql [hunk 1]
```diff
@@ -44,7 +44,7 @@ UNLOCK TABLES;
 -- Table structure for table `Email`
 --
 
-DROP TABLE IF EXISTS `Email`;
+DROP TABLE IF EXISTS `emails`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
 CREATE TABLE `Email` (
```
Comment (LUGAMAFE) [inline backup/Database.sql:50]:
```suggestion
CREATE TABLE `emails` (
```
Shouldn't it also say emails here?

File: backup/Database.sql [hunk 2]
```diff
@@ -53,7 +53,7 @@ CREATE TABLE `Email` (
   `information` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
   PRIMARY KEY (`id`),
   KEY `Correos_FK` (`idTypeEmail`),
-  CONSTRAINT `Correos_FK` FOREIGN KEY (`idTypeEmail`) REFERENCES `type_email` (`id`)
+  CONSTRAINT `Correos_FK` FOREIGN KEY (`idTypeEmail`) REFERENCES `email_types` (`id`)
```
Comment (LUGAMAFE) [inline backup/Database.sql:56]: what about 
```suggestion
  CONSTRAINT `emails_fk` FOREIGN KEY (`idTypeEmail`) REFERENCES `email_types` (`id`)
```

**Review by LUGAMAFE — CHANGES_REQUESTED** [review]

**Review by CheloChidoone — COMMENTED** [review]
Also remember to add the Labels for your PR.

**Review by LUGAMAFE — APPROVED** [review]

**Review by CheloChidoone — APPROVED** [review]

**Review by Cristianpan — CHANGES_REQUESTED** [review]
I haven't been able to test the backup because it gives me an error when importing. Please check if everything is correct and make the necessary changes.

[error](https://postimg.cc/FkKv0x4r)

**Review by CheloChidoone — APPROVED** [review]
It looks good and the importing works now, good job.

**Review by Cristianpan — APPROVED** [review]

**Review by FernandoJoachin — APPROVED** [review]

**Pull request merged**